manifests/proxydhcp.pp

```diff
@@ -31,14 +31,30 @@
     $failover = undef
   }
 
+  if !$foreman_proxy::dhcp_manage_acls {
```

Comment:
Nit: Can you flip it and write `if $foreman_proxy::dhcp_manage_acls { ... } else { ... }`? That feels a bit more natural to read.

Reply:
Done :+1: